test/e2e/cypress/fixtures/host-details/selected_host

```diff
@@ -1,9 +1,46 @@
 export const selectedHost = {
```

Comment:
I believe we are missing the file extension here `.js`

Reply:
Good catch! Fixed